### DISCUSSION TITLE
What is the difference between Test and Attempt

### DISCUSSION THREAD

**Discussion by AbdelAli-none** [codewars/codewars.com#3077] [Q&A]
Sometimes after I finish the kata and click on test, I find that the solution is correct, but when I click on attempt, I find that it is wrong.

**Comment by hobovsky** [comment]
It is explained in some places in documentation:
- https://docs.codewars.com/training/training-example#structure-of-a-kata
- https://docs.codewars.com/training/troubleshooting#test-attempt
- https://docs.codewars.com/authoring/tutorials/create-first-kata#code-snippets
- https://docs.codewars.com/authoring/guidelines/sample-tests and https://docs.codewars.com/authoring/guidelines/submission-tests
